mission_control/src/behaviors/depth_heading.cpp

```diff
@@ -91,7 +86,7 @@ BT::NodeStatus DepthHeadingBehavior::behaviorRunningProcess()
   else
   {
     ros::Duration delta_t = ros::Time::now() - behaviorStartTime_;
-    if (delta_t.toSec() > timeOut_)
+    if (delta_t.toSec() > timeOut_ && timeOutEnable_)
```

Comment:
@patapufate same comment as in https://github.com/ChrisScianna/ROS-Underwater-RnD/pull/112#discussion_r598710337.

Reply:
Fixed